lib/board.py

```diff
@@ -1,9 +1,24 @@
 class Board:
     def __init__(self):
-        pass
+        import itertools
+        ROWS = ['1', '2', '3', '4', '5', '6', '7', '8']
+        COLS = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+        RCtup = tuple(itertools.product(ROWS, COLS))
+        RCdic = {}
+        for i in RCtup:
+            if i == ('4', 'e') or i == ('5', 'd'):
+                RCdic[i] = "black"
+            elif i == ('4', 'd') or i == ('5', 'e'):
+                RCdic[i] = "white"
+            else:
+                RCdic[i] = None
+        self.RCdic = RCdic
 
     def get(self, row, col):
-        return 'white'
+        self.row = row
+        self.col = col
+        gettup = (self.row , self.col)
```

Comment:
```suggestion
        gettup = (row , col)
```

引数として渡される`row`と`col`はここでしか使わないので、インスタンス変数にしなくても良いかなと思いました。